### PR TITLE
Feature: Add DB validation

### PR DESCRIPTION
**Pull request by isaschm**
Addresses #8 and linting suggestions.

### CODE REVIEW

File: user_service/models/user_settings_model.py [hunk 1]
```diff
@@ -1,52 +1,19 @@
...
-        field_schema.update(type="integer")
-
 class UserSettingsSchema(BaseModel):
-    id : PyObjectId = Field(alias="_id", default_factory=PyObjectId) #TODO check if this is needed - maybe remove
+    id : str = Field(...)
     userId : int = Field(...)
     language: LanguageEnum = LanguageEnum.de #Default value is "de"
     #TODO more settings to be added here
-    
-    class Config:
```
Comment (akkbng) [inline user_service/models/user_settings_model.py:31]: @isaschm  Why are we removing config for UserSettingsSchema?

**Review by akkbng — COMMENTED** [review]
LGMT other than my question about the config in user settings

File: user_service/models/user_settings_model.py (hunk 1 above)
Comment (isaschm) [reply user_service/models/user_settings_model.py:31]: Ahhhhh that just looks like I removed the config! If you look at the whole file, you'll see it :)

Comment (akkbng) [reply user_service/models/user_settings_model.py:31]: ah sorry then, all good!

**Review by akkbng — APPROVED** [review]
LGTM

**Pull request merged**